### DISCUSSION TITLE
Image和Text的控制点位置不正确

### DISCUSSION THREAD

**Discussion by ye001226** [fabricjs/fabric.js#9743] [Q&A]
![1710823775449](https://github.com/fabricjs/fabric.js/assets/139834430/935f724b-83b5-42b7-b358-fa787f850070)
![1710823805727](https://github.com/fabricjs/fabric.js/assets/139834430/1be09d2b-8163-4e2e-9c47-4ce6a3f91693)
![1710823701793](https://github.com/fabricjs/fabric.js/assets/139834430/09b0e12c-fe3e-4a6c-8135-059d522d910f)
![1710823723872](https://github.com/fabricjs/fabric.js/assets/139834430/902d532c-e2c0-47e0-9cf8-af6ccf2fd7d8)
在我resize时，控制点有时候正确，有时候错误
![1710823950332](https://github.com/fabricjs/fabric.js/assets/139834430/235d3150-0539-42da-9f01-26afbde9f516)
![1710824009553](https://github.com/fabricjs/fabric.js/assets/139834430/334a503d-02ed-4c84-b7b9-fa7229a11659)

**Comment by ye001226** [comment]
version:5.3.0

**Comment by asturur** [comment]
hi @ye001226 the official language of the repository is english.
If you can't write english just double post your native language and an automated translation of your text.
Also all you added is a single line of explanation and 4 large screenshot.
That is not really the way to start any discussion.

**Reply by ye001226** [reply]
好的，我记住了，下次我会用你说的方式。
Okay, I'll keep that in mind. Next time I'll do it the way you said.
这个问题已经解决
The problem has been solved.
